harpy/_validations.py

```
@@ -11,14 +11,34 @@
+            f.read(10)
+        return True
+    except:
+        return False
+
+def is_plaintext(file_path):
+    """helper function to determine if a file is plaintext"""
+    try:
+        with open(file_path, 'r') as f:
+            f.read(10)
+        return True
+    except:
```

Ruff

Do not use bare `except`

(E722)